src/biomappings/resources/mappings.tsv

```diff
@@ -9037,6 +9046,7 @@ mondo	0019107	Rh deficiency syndrome	skos:exactMatch	mesh	C562717	Rh Deficiency
 mondo	0019155	Leydig cell hypoplasia	skos:exactMatch	mesh	C562567	Leydig Cell Hypoplasia	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
 mondo	0019169	pyruvate dehydrogenase deficiency	skos:exactMatch	mesh	D015325	Pyruvate Dehydrogenase Complex Deficiency Disease	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
 mondo	0019353	Stargardt disease	skos:exactMatch	mesh	D000080362	Stargardt Disease	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
+mondo	0019636	renal agenesis, unilateral	skos:exactMatch	mesh	D000075529	Solitary Kidney	semapv:ManualMappingCuration	orcid:0000-0002-1216-4761	semapv:LexicalMatching	generate_mondo_mesh_mappings.py	0.9
```

Comment:
the MONDO term appears to be specific towards the developmental condition whereas the MeSH term appears to be more about the effect

Reply:
@cthoyt yes i agree this is not an exact match as you mentioned.